dhall/tests/Dhall/Test/Tutorial.hs

```diff
@@ -33,7 +33,6 @@ tests =
         , Test.Tasty.testGroup "Unions"
             [ example 0 "./tests/tutorial/unions0A.dhall" "./tests/tutorial/unions0B.dhall"
             , example 1 "./tests/tutorial/unions1A.dhall" "./tests/tutorial/unions1B.dhall"
-            , example 2 "./tests/tutorial/unions2A.dhall" "./tests/tutorial/unions2B.dhall"
```

Comment:
How come BTW that we have tests for the tutorial both here and in https://github.com/dhall-lang/dhall-lang/tree/master/tests/normalization/success/haskell-tutorial?

Reply:
No good reason :slightly_smiling_face: